### DISCUSSION TITLE
Is it possible to specify a combination of controller buttons that would result in an action?

### DISCUSSION THREAD

**Discussion by adasium** [AntiMicroX/antimicrox#297] [Q&A] [answered]
I would like to use a trigger as a modifier, for example holding it down would allow faster mouse movement or right trigger + left stick up would result in increasing volume (while left stick without right trigger moves mouse cursor around)

**Comment by pktiuk** [comment, marked as answer]
No, AntiMicroX doesn't support this, but I think it is a cool idea.    
I created new issue with this request. https://github.com/AntiMicroX/antimicrox/issues/298 I will notify when it will be done (but don't expect it too soon)
